apps/tests/aws-runtime/test/test-service.ts

```diff
@@ -574,6 +575,11 @@ export const countersByNamespace = counter.index("countersOrderedByNamespace", {
   sort: ["namespace"],
 });
 
+export const countersByOptional2 = counter.index("countersByOptional2", {
+  partition: ["id"],
+  sort: ["optional", "n"],
+});
```

Comment:
Does the `optional` property get mapped to `string` when querying the index because it's guarnateed to be defined in the index?